app/scripts/modules/core/src/presentation/ToggleButtonGroupInput.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import { Tooltip } from './Tooltip';
+import { HelpField } from '../help';
+
+export interface IToggleButtonGroupInputProps {
+  propLabel: string;
+  propHelpFieldId?: string;
+  btnGroupSize?: string;
+  tooltipPropOnBtn?: string;
+  displayTextPropOnBtn?: string;
+  tooltipPropOffBtn?: string;
+  displayTextPropOffBtn?: string;
+  onClick: (value: boolean) => void;
+  isPropertyActive?: boolean;
+}
+
+export class ToggleButtonGroupInput extends React.Component<IToggleButtonGroupInputProps> {
```

Comment:
We can make this a functional component.

Reply:
`ToggleButtonGroupInput` shouldn't have been in the commit. I will remove this file. 
I replaced `ToggleButtonGroupInput ` from initial commit with `ToggleButtonGroup` which is a functional component. Sorry about that :#